### ISSUE TITLE
Alaska analysis failures

### ISSUE THREAD

**Issue by CloudNiner** [azavea/pfb-network-connectivity#423]
**Edit 2/8/2018:** Updated issue name, since all Alaska jobs fail with a similar error, it's not specific to Soldotna and Anchorage

Appears to be no scores, and thus one of the exports fails. Maybe no data because no OSM for this location?

Log below:
```

20:11:45
21 | population_total | 6582.0000 | | Total population of boundary

20:11:45
----+------------------------------------------+----------------+------------------+---------------------------------------------------------------------------------------------

20:11:45
1 | people | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this population score.

20:11:45
2 | opportunity_employment | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this employment score.

20:11:45
3 | opportunity_k12_education | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this K12 schools score.

20:11:45
4 | opportunity_technical_vocational_college | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this tech/vocational colleges score.

20:11:45
5 | opportunity_higher_education | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this universities score.

20:11:45
6 | opportunity | 0.0000 | 0.0000 |

20:11:45
7 | core_services_doctors | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this doctors score.

20:11:45
8 | core_services_dentists | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this dentists score.

20:11:45
9 | core_services_hospitals | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this hospital score.

20:11:45
10 | core_services_pharmacies | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this pharmacies score.

20:11:45
11 | core_services_grocery | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this grocery score.

20:11:45
12 | core_services_social_services | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this social services score.

20:11:45
13 | core_services | | |

20:11:45
14 | retail | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this retail score.

20:11:45
id | score_id | score_original | score_normalized | human_explanation

20:11:45
16 | recreation_trails | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this trails score.

20:11:45
17 | recreation_community_centers | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this community centers score.

20:11:45
18 | recreation | | |

20:11:45
19 | transit | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this transit score.

20:11:45
(21 rows)

20:11:45
15 | recreation_parks | 0.0000 | 0.0000 | On average, census blocks in the neighborhood received this parks score.

20:11:45
20 | overall_score | 0.0000 | 0.0000 |

20:11:45
Exporting analysis to /tmp/2901d9e6-4413-4263-9e55-927d11cdca5c/output/2901d9e6-4413-4263-9e55-927d11cdca5c

20:11:45
Updating job status: EXPORTING Exporting results

20:11:46
Initializing...

20:11:46
ERROR: Could not determine table metadata (empty table)

20:11:46
Updating job status: ERROR Failed See job logs for more details.
```

**Comment by spencerrecneps** [comment]
I'm traveling again today and tomorrow so I won't be able to look at this until Monday. I can't imagine it's a lack of data--Soldotna definitely has parks, a hospital, and some other features.

**Comment by CloudNiner** [comment]
Ok. We'll just communicate that these neighborhoods are currently unavailable. I think that's pretty reasonable since all the others finished without issues. We can run these separately whenever a fix appears.

**Comment by spencerrecneps** [comment]
@CloudNiner is it possible there's a projection problem? IIRC we're using UTM as the output, but maybe there's something about the projections of the input data? I know some projections are specific to the lower 48.

The other possibility is the Census might do things differently in AK. I'd have to dig into that a bit.

**Comment by CloudNiner** [comment]
I'm not sure off the top of my head what the root issue would be. Both of those things seem like good places to start, as we are now pulling in Alaska data via the geofabrik files. The jobs aren't missing source OSM data as far as I'm aware, although it may be related to missing features in the OSM data if its sparse enough?

**Comment by spencerrecneps** [comment]
This is still a mystery to me. I was able to complete a successful run locally, so it doesn't appear to be a problem with either census data or OSM.

I wonder if it could have something to do with the script that assigns the UTM projection. When I run it locally I'm able to assign the projection manually.

**Comment by spencerrecneps** [comment]
I think I've discovered the problem. It looks like the automatic UTM zone detection introduced in https://github.com/azavea/pfb-network-connectivity/issues/42 is assigning an incorrect SRID. I tried running the script on the attached shapefile from Anchorage and get SRID=3266 instead of the expected SRID=32606.

I'm pretty sure it's an issue with converting the integer to string that doesn't appear for the mainland because the UTMs are all two digit numbers that far east.

[ak.zip](https://github.com/azavea/pfb-network-connectivity/files/1729550/ak.zip)
